package.json

```diff
@@ -1,46 +1,37 @@
 {
   "name": "unified-language-server",
   "version": "0.3.0",
-  "description": "A language server for spoken languages using retext",
-  "homepage": "https://github.com/unifiedjs/unified-language-server",
+  "description": "Language server for content, using @unifiedjs",
```

Comment:
Does npm support `@` references? If not I suggest to remove it.

Anyway, this is not a blocker.